frontend/src/fixtures/personalScheduleFixtures.js

```diff
@@ -54,11 +77,11 @@ const personalScheduleFixtures = {
             "hostedDomain": "ucsb.edu",
             "admin": true
           },
-          "description": "DATA STRUCT ALGOR",
-          "quarter": "20221",
-          "name": "CS130A"
+          "description": "My Plan for Fall",
+          "quarter": "20229",
```

Comment:
20229 isn't a valid quarter... the last digit should be 1 for Winter, 2 for Spring, 3 for Summer, and 4 for Fall.   I think you probably want to change this to 20224